cypress/TestCases/FireboltCertification/UserGrants.feature

```diff
@@ -0,0 +1,788 @@
+            | Validate userGrants.request by denying for role use    | request grant for account uid capability with role use    | denied for account uid capability with role use    |
+            | Validate userGrants.request by denying for role manage | request grant for account uid capability with role manage | denied for account uid capability with role manage |
+
+    # Call usergrants.request with securestorage for role use, but the grant policy has role manage, since role mismatch it should return null since no grantpolicy is matched
+    @Usergrants @coreSDK @sdk @transport @requiresPlatformImplementation
+    Scenario: UserGrants.request R.1.1 - Negative Scenario: Validate userGrants.request call with role use, but grantpolicy having manage capability
+        Given the environment has been set up for 'userGrants' tests
+        And Framework registers 'pinChallenge' test provider
+        When 1st party app invokes the 'Firebolt' API to 'request grant for secure storage capability with role use'
+        And 3rd party 'certification' app is launched
+        And '3rd party app' invokes the 'Firebolt' API to 'get stored value with scope as device and key as authTestTokenDevice'
+        Then 'Firebolt' platform responds with 'expected value for authTestTokenDevice stored data in securestorage'
+        When '3rd party app' invokes the 'Firebolt' API to 'get info of secure storage capability'
+        And '3rd party app' invokes the 'Firebolt' API to 'check if account uid capability is granted with role as use'
+        And '3rd party app' invokes the 'Firebolt' API to 'check if account uid capability is granted with role as manage'
+        Then 'Firebolt' platform responds to '1st party app' with 'granted for secure storage capability with role use'
+
+    @Usergrants @coreSDK @sdk @transport @requiresPlatformImplementation
+    Scenario: UserGrants.request R.1.2 - Positive Scenario: Validate UserGrants.request which grantpolicy is not present in manifest
+        Given the environment has been set up for 'userGrants' tests
+        When 1st party app invokes the 'Firebolt' API to 'request grant for device info capability with role use'
+        And 3rd party 'certification' app is launched
+        And '3rd party app' invokes the 'Firebolt' API to 'fetch device platform'
+        Then 'Firebolt' platform responds with 'expected device platform'
+        When '3rd party app' invokes the 'Firebolt' API to 'get info of device info capability'
+        And '3rd party app' invokes the 'Firebolt' API to 'check if device info capability is granted with role as use'
+        And '3rd party app' invokes the 'Firebolt' API to 'check if device info capability is granted with role as manage'
+        Then 'Firebolt' platform responds to '1st party app' with 'granted for device info capability with role use'
```

Comment:
what is this TC trying to achieve ? 
what does line 729, 728 validate ?
what does line 730 validate ? Is it expecting granted ? How do you confirm if it is expecting granted/null ? what does firebolt spec say ?

Reply:
This testcase is without grant policy in manifest, so it should have default values, mostly it is null or true something. when you check in the validation object it returns true for capability.granted, which is default value.
Updated the key name to **default values for device info capability with role use**